client/packages/common/src/ui/forms/JsonForms/api/hooks/document/useAllocateNumber.ts

```diff
@@ -0,0 +1,10 @@
+import { useMutation, useQueryClient } from '@openmsupply-client/common';
+import { useAllocateNumberApi } from '../utils/useAllocateNumberApi';
+
+export const useAllocateNumber = () => {
+  const queryClient = useQueryClient();
+  const api = useAllocateNumberApi();
+  return useMutation(api.allocateNumber, {
+    onSuccess: () => queryClient.invalidateQueries(api.keys.base()),
```

Comment:
I'm wondering if that's necessary. will it cache the result of a mutation? I suppose it can't hurt to have this in

Reply:
no I don't think it is, copy and paste...  I changed it and removed the keys as well.